tests-beta/android/MASVS-STORAGE/MASTG-TEST-0216.md

```diff
@@ -29,9 +29,7 @@ Regardless of which approach the app used, Android provides a way to start the b
 1. Start the device.
 2. Install an app on your device.
 3. Launch and use the app going through the various workflows while inputting sensitive data wherever you can.
-4. Run the backup daemon.
-    - (a) Run [Backup Manager](https://developer.android.com/identity/data/testingbackup#TestingBackup)
-    - (b) Run `adb backup` (Deprecated in Android 12. The behavior might differ between an emulator and a physical device).
+4. Perform a backup and restore of the app data (@MASTG-TECH-0128).
```

Comment:
Why do you link a deprecated technique here? Why not TECH-0127?

Reply:
Please check again: `MASTG-TECH-0128` includes:
- Using the Backup Manager (via ADB shell)
- Using ADB Backup (deprecated)
- Using Android Backup Extractor

In this test you can use the one you like the most.